pandas/core/frame.py

```diff
@@ -5847,11 +5847,19 @@ def groupby(
 
         Parameters
         ----------%s
-        index : str or object, optional
+        index : str or object or a list of the previous, optional
```

Comment:
list of str, this really takes object? what exactly does that mean

Reply:
oh, my bad, i copied the description from `values`, will update a bit to make it clearer